plugins/workspace-plugin/src/theia-commands.ts

```diff
@@ -204,6 +205,9 @@ export class TheiaImportZipCommand implements TheiaImportCommand {
             try {
                 // download
                 const wgetArgs = [this.locationURI!, '-O', this.zipfilePath];
+                if (fs.existsSync(SS_CRT_PATH)) {
```

Comment:
As commented in the [JIRA issue](https://issues.redhat.com/browse/CRW-728?focusedCommentId=14002634&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-14002634), It seems that `wget` doesn't correctly support proxies.
I would suggest using `curl` instead of `wget`.
This way you would also be able to add the certificate authority.
I tested it with proxies + tls + self-signed certificates, inside a `che-theia` container of a Che installation on a disconnected OS 4.3 installation, and it works great.

This would mainly give:
```typescript
const curlArgs = [this.locationURI!, '-sL', '--output', this.zipfilePath];
if (fs.existsSync(SS_CRT_PATH)) {
    curlArgs.push('--cacert ');
    curlArgs.push(SS_CRT_PATH);
}
await execute('curl', curlArgs);
```

@mmorhun Do you think it would be worth updating the PR ?

cc @nickboldt who also proposed to use `curl` instead.

Reply:
@davidfestal I do not mind using `curl` instead of `wget` and even like it if it fixes proxy issue.
But when I tested your suggestion it didn't work on my side. Probably, `curl` treats given certificate as only one trusted. So we should either be sure that the endpoint has ours self-signed certificate or just trust everyone. So, instead of `--cacert` we have to use `-k`.

Reply:
Sure. I understand. I was mainly thinking of the case we import zip files from the devfile registry in the offline use-case. But sure, zip import should also work for external zip projects.
I'll test the `curl` command with `-k` on my side with the proxy to ensure it's OK, and will confirm here.

Reply:
So I tested, and the using:

```bash
curl <url> -kL --output <local file>
```

Also works great with proxies

Reply:
BTW, why not add the `-k` in all cases, if we really want to be able to download any zip with self-signed certificate ?
However does it involve any security issue ?

Reply:
We may add `-k` for all cases, but actually we might need it only when self-signed certificate is used. So I just followed this logic.

Reply:
got it